modules/tinymce/src/themes/silver/main/ts/api/Events.ts

```diff
@@ -35,6 +35,10 @@ const fireResolveName = (editor: Editor, node: Node): EditorEvent<{ name: string
     target: node
   });
 
+const fireToggleToolbarDrawer = (editor: Editor, state: boolean): void => {
+  editor.dispatch('ToggleToolbarDrawer', { isToolbarDrawerToggled: state });
```

Comment:
I wonder if this should just be called `state`? As it'd make it consistent with other events, for example: https://github.com/tinymce/tinymce/blob/a5cddb59ab810756e436940708cc94ba075c95c6/modules/tinymce/src/plugins/fullscreen/main/ts/api/Events.ts#L4
Additionally we already know it's about the toolbar drawer so it's doubling up. If not `state` then maybe it should just be `toggled` or `isToggled` instead?

Reply:
Updated to `isToolbarDrawerToggled` to match the already existing API [here](https://github.com/tinymce/tinymce/pull/8238/commits/691d7d20e19563bf30884ca99cba3aed9f66c90c).

Reply:
@yacodes you appear to have gone the wrong direction here? You're link isn't working, so I'm not sure what you were linking to, but as noted above we shouldn't be using `ToolbarDrawer` in the property name as that's already the event name. Also it would be more consistent to use `state` here instead as we already do elsewhere.